cypress/e2e/builder/enterSettings.cy.ts

```diff
@@ -112,10 +119,23 @@ describe('Load Settings', () => {
       MOCK_APP_SETTINGS[0].data.text,
     );
 
+    cy.get(buildDataCy(INITIAL_PROMPT_INPUT_FIELD_CY)).should(
+      'contain',
+      MOCK_APP_SETTINGS[2].data.text,
```

Comment:
You can access to your settings by name (using `.find`), this will be more robust than indexing